### ISSUE TITLE
Nostd support?

### ISSUE THREAD

**Issue by kaimast** [image-rs/image#1762]
It seems like this library requires std. Is there support for or plans for supporting compilation without the standard library?

**Comment by HeroicKatora** [comment]
No, nothing concrete. There is too much usage `std::io`, file system, allocation for this to be feasible. Maybe some of the dependencies can do this.

**Comment by 197g** [comment]
Duplicate of #1184 which in the meantime consolidated a few more comments on what is and was actually holding this back.
